shared/src/packages/users/libs/validation-schemas/user-sign-in.validation-schema.ts

```diff
@@ -0,0 +1,27 @@
+import joi from 'joi';
+
+import { UserValidationMessage } from '../enums/enums.js';
+import { type UserSignInRequestDto } from '../types/types.js';
+
+const userSignIn = joi.object<UserSignInRequestDto, true>({
```

Comment:
Please contact with sign up developer, we need to create a common schema for sign in and sign up, so please create the same one to prevent merge complex conflicts)

Reply:
@v0ldemar01 Sign up validation schema includes fullName, that should be required. If I use the same validation in sign in defo without full name, as I understand it just won't pass it. If it's needed to be done, can you explain how it should be implemented?

Reply:
So we will have a kind of common field schema with email and password and reuse it in sign up and sign in

Reply:
@v0ldemar01 I understand, however sign up validation apart from email and password should contain validation of full name (otherwise client will be able to enter empty full name), while sign in validation shouldn't contain this field. At now these validation are lookng the same at my branch, but it's because I didn't extend sign up validation. There's sign up validation made by @Ekit18 in his issue #15:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/2c360def-0406-41ed-bbef-00b2a7037bb1)

Reply:
we will use different schemas, it's ok, because they will have different properties

Reply:
we will use different schemas, it's ok, because they will have different properties